src/packagedcode/pypi.py

```diff
@@ -112,6 +112,8 @@ def parse(location):
         file_name = fileutils.file_name(location)
         parsers = {
             'setup.py': parse_setup_py,
+            'requirements.txt': parse_requirements_txt,
+            'requirements.in': parse_requirements_txt,
```

Comment:
Also note that for the future, I have submitted this PR upstream https://github.com/pyupio/dparse/pull/48

Reply:
:+1: